README.md

```diff
@@ -87,11 +87,23 @@ $ curl -H "Content-Type: application/json" \
 ```
 
 You can then update the resource with a **PATCH** request:
+
+Note: in the example file, the `externalIdentifier` and `identification.identifier` must be set to the value returned from the POST, and
+the `tacoIdentifier` and `identification.sdrUUID` must be set to the value retrieved in the GET
```

Comment:
I'm not sure this is true about `identification.sdrUUID` is it?

Reply:
shit, good catch. no, that is not correct. 

* externalIdentifier == identification.identifier
* tacoIdentifier != identification.sdrUUID
* identification.sdrUUID == UUID assigned by *fedora 3 / DOR*, which we cannot prove its uniqueness.

Reply:
Whoops, mistake on my part here, will fix.